rskj-core/src/test/java/co/rsk/net/light/LightSyncStateTest.java

```diff
@@ -283,8 +292,9 @@ public void peerInStartRoundGetLesserHeadersThanAsked() {
 
         newBlockHeadersInStartRoundSyncState(startBlockHeader, blockHeaders, longLightPeer);
 
-        verify(lightSyncProcessor, times(1)).failedAttempt();
+        verify(lightSyncProcessor, times(1)).failedAttempt(longLightPeer);
         assertEquals(lightSyncProcessor.getSyncState().getClass(), StartRoundSyncState.class);
+        assertThat(lightPeersInformation.getConnectedPeersSize(), is(1));
     }
 
```

Comment:
What's happen if a peer connects to the lightpeer, then it has a bad behavior, the process aborts, and then the peer tries to connect again ?

Reply:
This should be taken into account in the Timeouts system, mainly because the peer should be banned for a period of time at least after some tries